tests-functional/tests/test_local_pairing.py

```diff
@@ -219,17 +219,14 @@ def test_pairing_server_as_sender(self):
         assert bob_response is True
 
         # Check that the messages are synced
-        # TODO uncomment this when message sync is enabled (syncMessages = true)
-        # those asserts are just to avoid unused variable warnings for now
-        assert message_id is not None
-        assert sender_chat_id is not None
-        # messages = bob_second_device.wakuext_service.chat_messages(sender_chat_id, limit=10)["messages"]
-        # message_found = False
-        # for message in messages:
-        #     if message["id"] == message_id and message["text"] == "test_message":
-        #         message_found = True
-        #         break
-        # assert message_found, "Message sent before pairing not found on paired device"
+        messages = bob_second_device.wakuext_service.chat_messages(sender_chat_id, limit=10)["messages"]
+        assert messages is not None, "No messages found on paired device"
+        message_found = False
+        for message in messages:
+            if message["id"] == message_id and message["text"] == "test_message":
+                message_found = True
+                break
```

Comment:
some python shenanigans could be applied 😄 
```suggestion
        message_found = any(msg["id"] == message_id and msg["text"] == "test_message" for msg in messages)
```